(docs) Add list of external plugins (#1709)
puppetlabs/bolt: resources/plugins.md

> # 3rd Party Bolt Plugin list

Reviewer: ```suggestion
# Bolt third-party plugins
```

> Below is a plugin list of all documented 3rd party bolt plugins.  If you have created a plugin please add your plugin

Reviewer: ```suggestion
Below is a list of documented third-party plugins for Bolt.  If you have created a plugin you can add it to the list and [open a pull request](https://github.com/puppetlabs/bolt/blob/master/CONTRIBUTING.md#pull-requests) with the changes.
```

> to the resources/plugins.yaml list and run the rake task `plugin_list` to regenerate the list with your info.  Submit a PR with the updated files so others can see your plugin information. 

Reviewer: ```suggestion
```

> ## Bolt Plugin Table

Reviewer: ```suggestion
## Plugins
```

> |Name|Url|Author|Category|Description|Tags|Requirements|

Reviewer: ```suggestion
| Plugin | Description | Documentation | Author |
```

> | --- | --- | --- | --- | --- | --- | --- |

Reviewer: ```suggestion
| --- | --- | --- | --- |
```

> |AD Inventory |https://github.com/glennsarti/bolt_ad_inventory |glennsarti |inventory |Bolt Plugin for getting inventory from Active Directory |windows, Active Directory, Inventory |bolt 1.x|

Reviewer: ```suggestion
| `ad_inventory` | Generate targets from Windows Active Directory queries. | [bolt_ad_inventory](https://github.com/glennsarti/bolt_ad_inventory) | [glennsarti](https://github.com/glennsarti) |
```
